csrc/xpu/grouped_gemm/xe_default/grouped_gemm_xe_default.cpp

```diff
@@ -0,0 +1,31 @@
+#include <torch/all.h>
+#include "grouped_gemm_xe_default.h"
+#include "grouped_gemm.hpp"
+
+torch::Tensor cutlass_grouped_gemm_xe_default(
+    torch::Tensor ptr_A,
+    torch::Tensor ptr_B,
+    const c10::optional<at::Tensor>& ptr_bias,
+    torch::Tensor ptr_D,
+    torch::Tensor expert_first_token_offset,
+    int64_t N,
+    int64_t K,
+    int64_t num_experts) {
+  auto ptr_bias_ = ptr_bias;
+  if (ptr_bias.has_value()) {
+    auto expert_token_count = (expert_first_token_offset.slice(0, 1) -
```

Comment:
cc @Liangliang-Ma I try to convert repeat bias part into torch C++ API. please help check correctness...